docs/_documentations/performance.md

```diff
@@ -28,7 +28,20 @@ You will learn how to add and enable monitoring to your application. Then, you c
 1. From your IDE, go to the Codewind plug-in and right-click on a project.
 2. Then, select **Performance Dashboard**. A window appears that displays the Performance Dashboard.<br>
 
-![performance dashboard](images/performanceguide/performancedash_window.png){:height="375px" width="800px"}<br>
+![Performance dashboard](images/performanceguide/performancedash_window.png){:height="375px" width="800px"}<br>
+
+## Viewing the status of your project
+
+1\. View the status of your project, and the performance dashboard features available to it, in the **Capabilities** pane:
+
+![Performance dashboard capabilities](images/performanceguide/performance_dashboard_capabilities.png)
+
+- **Project status:** Provides current project status.
+- **Run Load Feature:** Provides information about whether load pressure can be applied to your project.
+- **Live monitoring:** Use this information to determine whether you can view live metrics. For example, Appsody projects protect the metrics endpoint by requiring authentication. Authentication can stop the performance dashboard from accessing the endpoint and from being able to retrieve performance metrics. You can programmatically resolve disabling authentication by following the suggestions that are provided, or you can click **Disable authentication** and Codewind disables authentication for you. When you click **Disable authentication**, you see that the project status updates and messages appear in the **Notification** pane. You can stack the notifications, remove them after a few seconds, or close them. 
+- **Benchmarks:** Benchmark your project as you run your tests. The charts display comparisons across load runs.
```

Comment:
Exactly,   Benchmarks are a collection of completed load test results displayed on a chart and in a table.

Reply:
Changed to your suggestion.